### PR TITLE
statichit: misc. fixes

### PR DESCRIPTION
**Pull request by randall**
Anchor request to remapped path and allow for relative --file-paths

### CODE REVIEW

File: plugins/experimental/statichit/statichit.cc [hunk 1]
```diff
@@ -571,6 +571,14 @@ TSRemapDoRemap(void *ih, TSHttpTxn rh, TSRemapRequestInfo *rri)
     return TSREMAP_NO_REMAP;
   }
 
+  // Anchor to URL specified in remap
+  int pathsz;
+  TSUrlPathGet(rri->requestBufp, rri->requestUrl, &pathsz);
+  if (pathsz > 0) {
+    TSHttpTxnStatusSet(rh, TS_HTTP_STATUS_NOT_FOUND);
```
Comment (shukitchan) [inline plugins/experimental/statichit/statichit.cc:578]: Perhaps add an error message - VERROR("...");

**Review by shukitchan — APPROVED** [review]
looks trivial enough. 
just some nitpicking comments

File: plugins/experimental/statichit/statichit.cc (hunk 1 above)
Comment (randall) [reply plugins/experimental/statichit/statichit.cc:579]: Thanks @shukitchan ! Added a VERROR

**Review by shukitchan — APPROVED** [review]

**Comment by randall** [comment]
[approve ci freebsd]

**Comment by randall** [comment]
[approve ci freebsd]

**Comment by zwoop** [comment]
Cherry-picked to v9.0.x branch.
Cherry-picked to v9.1.x branch.

**Pull request merged**